GliaWidgets/Sources/ViewModel/Chat/ChatViewModel.Environment.Mock.swift

```diff
@@ -31,7 +31,14 @@ extension ChatViewModel.Environment {
         createFileUploadListModel: SecureConversations.FileUploadListViewModel.mock(environment:),
         createSendMessagePayload: { _, _ in .mock() },
         proximityManager: .mock,
-        log: .mock
+        log: .mock,
+        operatorRequestHandlerService: .init(
```

Comment:
The same here

Reply:
fixed [3cdd827](https://github.com/salemove/ios-sdk-widgets/pull/885/commits/3cdd827dfc9ce2142ffb58695df997af74b1be60)